evcache-core/src/test/java/com/netflix/evcache/test/SimpleEVCacheTest.java

```diff
@@ -32,8 +34,8 @@
 public class SimpleEVCacheTest extends Base {
     private static final Logger log = LogManager.getLogger(SimpleEVCacheTest.class);
 
-    private static final String APP_NAME = "EVCACHE_TEST";
-    private static final String ALIAS_APP_NAME = "EVCACHE";
+    private static final String APP_NAME = "EVCACHE_TEST_SRIRAM";
```

Comment:
if this warrants needing a real cluster, it maybe useful to have (and refer) a generic integration test cluster (like evcache_cde_integration_test)